pkg/registry/apis/dashboard/legacy/sql_dashboards.go

```diff
@@ -69,72 +75,100 @@ func NewDashboardAccess(sql db.DB, namespacer request.NamespaceMapper, dashStore
 
 const selector = `SELECT
 	dashboard.org_id, dashboard.id,
-	dashboard.uid,slug,
-	dashboard.folder_uid,
-	dashboard.created,dashboard.created_by,CreatedUSER.login,
-	dashboard.updated,dashboard.updated_by,UpdatedUSER.login,
-	plugin_id,
+	dashboard.uid, dashboard.folder_uid,
+	dashboard.created,CreatedUSER.uid as created_by,
+	dashboard.updated,UpdatedUSER.uid as updated_by,
+	dashboard.deleted, plugin_id,
 	dashboard_provisioning.name as origin_name,
 	dashboard_provisioning.external_id as origin_path,
 	dashboard_provisioning.check_sum as origin_key,
 	dashboard_provisioning.updated as origin_ts,
-	dashboard.version,
-	title,
-	dashboard.data
+	dashboard.version, '', dashboard.data
   FROM dashboard
   LEFT OUTER JOIN dashboard_provisioning ON dashboard.id = dashboard_provisioning.dashboard_id
   LEFT OUTER JOIN user AS CreatedUSER ON dashboard.created_by = CreatedUSER.id
-  LEFT OUTER JOIN user AS UpdatedUSER ON dashboard.created_by = UpdatedUSER.id
-  WHERE is_folder = false`
+  LEFT OUTER JOIN user AS UpdatedUSER ON dashboard.updated_by = UpdatedUSER.id
+  WHERE dashboard.is_folder = false`
 
-func (a *dashboardSqlAccess) getRows(ctx context.Context, query *DashboardQuery, onlySummary bool) (*rowsWrapper, int, error) {
-	if !query.Labels.Empty() {
-		return nil, 0, fmt.Errorf("label selection not yet supported")
-	}
-	if len(query.Requirements.SortBy) > 0 {
-		return nil, 0, fmt.Errorf("sorting not yet supported")
-	}
-	if query.Requirements.ListHistory != "" {
-		return nil, 0, fmt.Errorf("ListHistory not yet supported")
-	}
-	if query.Requirements.ListDeleted {
-		return nil, 0, fmt.Errorf("ListDeleted not yet supported")
+const history = `SELECT
+	dashboard.org_id, dashboard.id,
+	dashboard.uid, dashboard.folder_uid,
+	dashboard_version.created,CreatedUSER.uid as created_by,
```

Comment:
should we use the dashboard creator here and just use the version creator as the updated by?